### PR TITLE
funny names

### PR DESCRIPTION
**Pull request by gwprice115**
give all characters whimsical names

https://user-images.githubusercontent.com/4099764/182274465-10f78a97-dde9-4eeb-ba5f-c59f7d853e7f.mov

### CODE REVIEW

File: server/player/nameGenerator.ts [hunk 1]
```diff
@@ -0,0 +1,482 @@
+import { getRandomElement } from "../utils.js";
+
+const USED_NAMES = new Set<string>();
```
Comment (gwprice115) [inline server/player/nameGenerator.ts]: crap this is a bug

Comment (gwprice115) [reply server/player/nameGenerator.ts]: fixed

File: client/src/scenes/game.ts [hunk 2]
```diff
@@ -168,10 +168,18 @@ export class GameScene extends Phaser.Scene {
     });
   }
 
-  private addPlayer({ id, x, y }: Player) {
+  private addPlayer({ id, x, y, name }: Player) {
     const sprite = new Phaser.GameObjects.Sprite(this, x, y, "player").setOrigin(0.5, 1);
+    const nameText = new Phaser.GameObjects.Text(this, x, y, name, {
```
Comment (hpx7) [inline client/src/scenes/game.ts]: don't you want `y - sprite.height` or something?

Comment (gwprice115) [reply client/src/scenes/game.ts]: yea

**Review by hpx7 — APPROVED** [review]

**Pull request merged**